Memorize/Screens/ThemeEditor/ThemeEditorView.swift

```diff
@@ -3,6 +3,22 @@ import SwiftUI
 struct ThemeEditorView: View {
     var theme: ThemeEditorViewModel
```

Comment:
it seems that this is not `theme` anymore and it's `viewModel`

Reply:
agree